scripts/build/with-react.js

```diff
@@ -0,0 +1,13 @@
+import '../../lib/ckeditor/ckeditor';
+
+import * as AlloyEditor from '../../src/adapter/main';
+import React from 'react';
+
+if (typeof window !== 'undefined') {
+	window.CKEDITOR.disableAutoInline = true;
+	window.AlloyEditor = AlloyEditor;
+	window.AlloyEditor.React = React;
+}
+
+export {AlloyEditor, React};
```

Comment:
Do we need to export `React` here?

Reply:
We don't ... but we had fun testing this thanks!